packages/server/README.md

```diff
@@ -20,21 +20,49 @@ This will start the server, which will listen on port **22300** on **localhost**
 ## Setup the database
 
 You can setup the container to either use an existing PostgreSQL server, or connect it to a new one using docker-compose
-
 ### Using an existing PostgreSQL server
 
-To use an existing PostgresSQL server, set the following environment variables in the .env file:
+To use an existing PostgresSQL server you must configure some environment variables in the .env file: 
 
+First set the `DB_CLIENT` environment variable to `pg`
 ```conf
 DB_CLIENT=pg
+```
+
+#### Defining the connection
+Use a connection string variable:
```

Comment:
Could you link to a page that shows the list of available parameters? (ideally on the official Postgres website)